ProcessLib/SmallDeformationWithLIE/Common/LevelSetFunction.cpp

```diff
@@ -0,0 +1,42 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ */
+
+#include "LevelSetFunction.h"
+
+#ifndef Q_MOC_RUN // to avoid Qt4 bug, https://bugreports.qt.io/browse/QTBUG-22829
+#include <boost/math/special_functions/sign.hpp>
+#endif
+
+#include "FractureProperty.h"
+
+namespace
+{
+// Heaviside step function
+inline double Heaviside(double v)
+{
+    return (v < 0.0) ? 0.0 : 1.0;
+}
+
+} // no named namespace
+
+namespace ProcessLib
+{
+namespace SmallDeformationWithLIE
+{
+
+double calculateLevelSetFunction(
+        FractureProperty const& frac, double const* x_)
```

Comment:
it's possible but not better, because the function will be used with MathLib::Point3d.